README.md

```diff
@@ -239,6 +239,7 @@ Parameters for this backend:
 | filter           | Filter the users, eg "(&(memberOf=CN=group,OU=Users,OU=Company,DC=example,DC=com)(objectClass=user)(sAMAccountName=%s))" |
 | principal_suffix | suffix to append to usernames (eg: @example.com)                                                                         |
 | pool_size        | size of the connection pool, default is 10                                                                               |
+| authenticated    | true to force password be filled even server allow UNAUTHENTICATED login                                                 |
```

Comment:
Maybe the this should be unauthenticated instead of authenticated as it's confusing

Reply:
Currently the plugin already allow unauthenticated login. I put the parameter to change this behavior enforcing authenticated policy.

If I invert this logic, turn parameter on will keep current behavior, turn parameter off will change the behavior.

I don't know if inform parameter off to enable function is clear but if you think it is better I can change.